frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -21,9 +21,18 @@ import {
 } from '@patternfly/react-core';
 
 import { history } from './router';
+import { Dropdown } from '../utils';
 
-const CATEGORY_URL_PARAM = 'category';
-const KEYWORD_URL_PARAM = 'keyword';
+export const FilterType = {
+  category: 'category',
+  keyword: 'keyword',
+  groupBy: 'groupBy',
```

Comment:
No need to use `groupBy` everywhere.
```suggestion
  group: 'group',
```

Reply:
Grouping should not be a filter.